conf/messages

```diff
@@ -199,18 +201,18 @@ obligations.dropdown.declarations.li2.type2.v2                     = Final check
 obligations.dropdown.declarations.li1.text.v2                      = - Depending on the end date of your entity’s accounting period, you will need to submit a declaration through your software. This is to confirm all of the returns you submitted during the course of your entity’s accounting period were true to your knowledge.
 obligations.dropdown.declarations.li2.text.v2                      = - This is to confirm that you have disclosed your income from all sources true to your knowledge. You need to do send this declaration using your software.
 obligation.dropdown.quarterlyReturn.text.v2                        = A quarterly update sets out the income and expenses for each 3-month period.
-obligation.dropdown.quarterlyReturn.text.v2.lin2.prt1              = Using your
+obligation.dropdown.quarterlyReturn.text.v2.lin2                   = Using your {0}, you must submit 4 quarterly updates in a year for each source of income.
```

Comment:
I see what you have done combining the models, but you should probably remove the prt2 of the messages you have done this for